packages/dev-server/README.md

```diff
@@ -28,6 +28,13 @@ module.exports = createConfig([
 ```
 
 
+## Pitfalls
+
+Make sure you use this `dev-server` block after your `entryPoint()` blocks. The dev server will extend all of them with the `'webpack/hot/only-dev-server'` entry point by default.
```

Comment:
That's right, the order is not important.

But I mean something else: `dev-server` adds itself to every `entry` key it finds in the config. If you define entry points (chunk name !== 'main') after `dev-server` then `dev-server` will not have added the dev-server-entry to this later entry's key.